Documentation/getting_started/1_creating_a_new_project_vs.md

```diff
@@ -2,7 +2,7 @@ Start up Visual Studio and select **New Project...** in the upper left corner.
 
 ![New Solution](images/getting_started/1_new_soulution_vs.png)
 
-Now you should see a "New Project" dialog pop up. From here select **Templates > Visual C# > MonoGame** category, then select **MonoGame Cross Platform Desktop Project**. Next type in the name that you wish to give your project. Do note that you should not use space character for it. For this tutorial, it will be named **ExampleGame**. After you've entered the name, click on the **Browse** button next to location text field, and select where you wish to save your project. Finally click **OK** to create a new project.
+Now you should see a "New Project" dialog pop up, from here select **Templates > Visual C# > MonoGame** category, and then select **MonoGame Cross Platform Desktop Project**. Next type in the name that you wish to give your project, for this tutorial lets just use **ExampleGame** (do note that you should not use space character for it). After you've entered the name, click on the **Browse** button next to location text field, and select where you wish to save your project. Finally click **OK** to create a new project.
```

Comment:
lets -> let's
you should not use a space character for it -> awkward phrasing
button next to **the** location text field